### ISSUE TITLE
Hard Swish support in TFLite Micro

### ISSUE THREAD

**Issue by anferico** [tensorflow/tensorflow#41240]
@tensorflow/micro

While support for the HARD_SWISH operation has been added to TFLite Micro, there's no corresponding layer in TensorFlow's and Keras's APIs. 
After converting a TensorFlow/Keras model containing HARD_SWISH layers to TFLite, how do I instruct the TFLite interpreter to use the custom implementation of HARD_SWISH available in [tensorflow/lite/micro/kernels/hard_swish.cc](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/kernels/hard_swish.cc)?
I know that `hard_swish(x)` could be implemented using Keras's ReLU layer as `x * (relu6(x + 3) / 6)`, but I guess it wouldn't take advantage of its dedicated implementation for microcontrollers.

**Comment by njeffrie** [comment]
If I understand correctly, you have a TFLite model with the HARD_SWISH operator in it. In this case, simply running the same model on TF Micro should succeed, since operator registration works in the same way between Lite and Micro. When you build using the TF Micro Makefile (tensorflow/lite/micro/tools/make/Makefile) it will automatically include all TF Micro versions of kernels, rather than the TF Lite versions you get when you build for TF Lite.

If you do run the model and run into issues resolving the HARD_SWISH operator, or if the operator itself causes issues please re-open with an update or file an issue with those specific errors. If my assessment is incorrect, please feel free to re-open and clarify.

**Comment by YannPourcenoux** [comment]
I think he meant that to implement the hardswish in keras he uses MUL and ADD operations that get converted as such in TFLite although he would like to know how to create his keras model in order to get the hardswish operation in his TFLite file after conversion.
(Which is my current problem)
